cel/env.go

```diff
@@ -793,6 +796,16 @@ func (e *Env) configure(opts []EnvOption) (*Env, error) {
 		}
 	}
 
+	// Precalculate function bindings
+	e.functionBindings = []*functions.Overload{}
```

Comment:
Could you perhaps move the binding resolution into the `newProgram()` call as `sync.Once`? In some cases the bindings won't need to be resolved at all, so it would be great to defer this to where the program is actually built.

Reply:
Thank you!